includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -252,7 +252,9 @@ private static function buildDescriptorSettings(
 		foreach ( $config->get( 'ManageWikiSettings' ) as $name => $set ) {
 			$mwRequirements = $set['requires'] ? ManageWikiRequirements::process( $set['requires'], $extList, false, $wiki ) : true;
 
-			$add = ( isset( $set['requires']['visibility'] ) ? $mwRequirements : true ) && ( ( $set['from'] == 'mediawiki' ) || ( in_array( $set['from'], $extList ) ) );
+			$privateConfig = isset( $set['private'] ) && $set['private'];
```

Comment:
sounds like this would either;

a) a good required variables, or
b) a 'requires' sub param

Reply:
I think I did it by making it a sub part of the visibility key.